org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -230,14 +230,22 @@ Connection:
     | ( '(' leftPorts += VarRef (',' leftPorts += VarRef)* ')' isIterated ?= '+'?))
     ('->' | physical?='~>') 
     rightPorts += VarRef (',' rightPorts += VarRef)*
-    (delay=Delay | ';'?);
+    (delay=Delay)?
+    (serialization=Serialization)?
```

Comment:
I would call this attribute `serializer` (and the production/class `Serializer`). That way we have `.getSerializer().getName()` which reads better.

Reply:
Alternatively, we could use `serialization` and `method`. Either works for me.

Reply:
Apparently, `name` in XText has to be unique. That means if I add two or more `serializer`s on different connections, I get the following error...

    ^^^^^^ Duplicate Serializer 'ros2'